cloud/passportauth/src/auth/PassportAuth.ts

```diff
@@ -107,10 +108,10 @@ export class PassportAuth implements Auth {
    *
    * @param passport - passport.js instance
    */
-  protected setup(passport: passport.Passport) {
-    passport.use(new Strategy(defaultStrategy(this.userSec)));
-    passport.serializeUser(defaultSerializeUser);
-    passport.deserializeUser(defaultDeserializeUser);
+  protected setup(passportApi: passport.Passport) {
+    passportApi.use(new Strategy(defaultStrategy(this.userSec)));
+    passportApi.serializeUser(defaultSerializeUser);
+    passportApi.deserializeUser(defaultDeserializeUser);
```

Comment:
yeah, we can alias the `import * as passportApi from 'passport'` and keep the parameter with the prettier name, because the latter is expected to be seen from outside the module.